### PR TITLE
Always lay down pip.conf on RPCD plays

### PR DESCRIPTION
**Pull request by BjoernT**
This fix ensures that a proper pip.conf is layed down
before installing RPC-O components.
This furthermore ensures that upon deploy/upgrade.sh script errors
manual continued playbook installation will install the right pip
packages based on the correct OSA version.

Closes-Bug: #1391

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/staging.horse/build-14a0674fa5/docs/private-cloud/release-notes/).

**Comment by andymcc** [comment]
👎  Add these as dependencies on the role (in meta/main.yml - for example: https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/kibana/meta/main.yml#L30-L31 )
Additionally, the setup-logging.yml is unrequired, since kibana and elasticsearch already depend on pip_lock_down, and logstash has no pip packages.
The MaaS dependency is being changed here: https://github.com/rcbops/rpc-openstack/pull/1405

TL;DR just add the dependency to meta/main.yml for rpc_support and horizon_extension roles.

**Comment by BjoernT** [comment]
Closing in favor of other PR
